### DISCUSSION TITLE
Can't find a way to pick up the current URL

### DISCUSSION THREAD

**Discussion by rumemo** [Rafostar/clapper#483] [General]
I've been using Clapper, now, for a couple of weeks. Other than for a minor issue that I've reported, I find it to be an amazingly useful player.

However, **I haven't found any way to view or grab the URL of the video that's currently playing.** The playlist of titles is clearly displayed, of course, (Also, I did see a reference to a feature on an older version of Clapper, but it didn't work on the version I use.)

Would anyone happen to know of such a feature?

Thanks in advance for any help.

I'm running Version 0.6.1 under Cinnamon Mint 21.3.

Troubleshooting info:

GLib 2.78.6
GTK 4.12.5
Adwaita 1.4.7
GStreamer 1.22.8

**Comment by Rafostar** [comment]
There is currently no such functionality to take URI from queue (the other way around). Nobody requested for this yet :smile:. If you need it, please open a feature request in issues tab on GitHub.
